docs/root/start/sandboxes/route-mirror.rst

```diff
@@ -0,0 +1,146 @@
+.. _install_sandboxes_route_mirror:
+
+Route mirroring policies
+========================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+This simple example demonstrates Envoy's request mirroring capability using
+`request mirror policies <https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-routeaction-requestmirrorpolicy>`__.
+
+Incoming requests are received by ``envoy-front-proxy`` service. The envoy instance running
+in this container is configured with two routes (:download:`front-envoy.yaml <_include/route-mirror/front-envoy.yaml>`):
+
+.. literalinclude:: _include/route-mirror/front-envoy.yaml
```

Comment:
im not sure about putting the example code here - i think it distracts a bit from the explanation (and doesnt work so well with formatting in at least my browser)

im wondering whether we should keep it and/or whether we might want to put it in the corresponding steps - ie show the static config in the static config step, etc

Reply:
I took a stab at inlining it. for example:

![image](https://user-images.githubusercontent.com/76932416/197695720-2178b2a1-f28a-40ac-b425-c1ea62eba78a.png)